### PR TITLE
New events: Women In Federal IT & Cyber Conference Panel Discussion #…

### PR DESCRIPTION
**Pull request by ayerkes**
…2: Expanding Our Network

This PR implements the following **changes:**

1. creating the event page for the Women in IT event.

**URL / Link to page**

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/Women-in-IT-July-event/event/2020/07/16/women-in-federal-it-cyber/

### CODE REVIEW

File: content/events/2020/07/2020-07-16-women-in-federal-it-cyber.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# Page weight: controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+weight: 0
+
+# Make it better ♥
+
+---
+
+## Presenters:
```
Comment (saracope) [inline content/events/2020/07/2020-07-16-women-in-federal-it-cyber.md:39]: I think the Presenters title should go a bit further down after the intro @ayerkes

**Comment by saracope** [comment]
@ayerkes Will these folks be added as authors?

File: content/events/2020/07/2020-07-16-women-in-federal-it-cyber.md (hunk 1 above)
Comment (ayerkes) [reply content/events/2020/07/2020-07-16-women-in-federal-it-cyber.md:39]: Okay I will update!

**Comment by ayerkes** [comment]
> @ayerkes Will these folks be added as authors?

@saracope yes @DanielJPino-ux added them, I just need to move them over to this PR now.

**Comment by ayerkes** [comment]
> > @ayerkes Will these folks be added as authors?
> 
> @saracope yes @DanielJPino-ux added them, I just checked and they need to be merged.

**Comment by saracope** [comment]
Ok, I think I just merged in the last author. Do you want to update here or did you plan on adding a new PR? @ayerkes @DanielJPino-ux

**Comment by ayerkes** [comment]
@saracope 

> Ok, I think I just merged in the last author. Do you want to update here or did you plan on adding a new PR? @ayerkes @DanielJPino-ux

@saracope I will add them directly to this PR now.

**Pull request merged**